mainnet/run-observer.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
-docker run --mount type=bind,source=${OBSERVER_DIR}/db,destination=/go/elrond-go/cmd/node/db --mount type=bind,source=${OBSERVER_DIR}/logs,destination=/go/elrond-go/cmd/node/logs --mount type=bind,source=${OBSERVER_DIR}/config,destination=/config --publish ${P2P_PORT}:37373 --network=elrond-squad --ip=${IP} --name squad-${SHARD} elrondnetwork/elrond-node-obs:v1.3.42 \
+docker run --mount type=bind,source=${OBSERVER_DIR}/db,destination=/go/elrond-go/cmd/node/db --mount type=bind,source=${OBSERVER_DIR}/logs,destination=/go/elrond-go/cmd/node/logs --mount type=bind,source=${OBSERVER_DIR}/config,destination=/config --publish ${P2P_PORT}:37373 --network=elrond-squad --ip=${IP} --name squad-${SHARD} elrondnetwork/elrond-node-obs:v1.3.44 \
```

Comment:
not necessarily in this PR, but i think that multi line commands will be useful

Reply:
it's a task for future @miiu96